packages/fluent-theme/src/components/linerActivity/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as isLinerMessageActivity } from './private/isLinerMessageActivity';
```

Comment:
If they are exposing publicly, should we move them from `./private/` to `./` instead?

Reply:
Maybe, I'm not sure regarding the convention :rofl:

Reply:
It's fine. The convention is... we *could* write a ESLint forbidding people from importing from private from another feature, unless it's explicitly allowed. On in the hands, `../**/private/**` is forbidden, but `./**/private/**` is good.

Second-thought, if we have the barrel file (`index.ts`), everything we want to expose will be inside the barrel file.

Does it mean we should write a ESLint rule to force everyone importing `../**/index` or `./**` only?

Unsure... the broader idea is, don't import what we did not export. As long as we can write a ESLint rule to fulfill this goal, any approach is fine.